integration/testutil.go

```diff
@@ -251,7 +251,7 @@ services:
     - "lazy-containerd-data:/var/lib/containerd"
     - "lazy-soci-snapshotter-grpc-data:/var/lib/soci-snapshotter-grpc"
   registry:
-    image: ghcr.io/oras-project/registry:v1.0.0-rc
+    image: ghcr.io/oci-playground/registry:latest
```

Comment:
Can you use another tag, other than latest, so we have repeatability?

Reply:
Done.